stl/inc/format

```diff
@@ -191,6 +212,31 @@ constexpr const _CharT* _Parse_width(const _CharT* _Begin, const _CharT* _End, _
     return _Begin;
 }
 
+template <class _CharT, _Parse_spec_callbacks<_CharT> _Callbacks_type>
+constexpr const _CharT* _Parse_precision(const _CharT* _Begin, const _CharT* _End, _Callbacks_type&& _Callbacks) {
+    ++_Begin;

```

Comment:
Should we `_STL_INTERNAL_CHECK(_Begin != _End && *_Begin == '.');` here? I recall validation for such preconditions in the first PR.

Reply:
nope, _Parse_precision (and the parse functions for other parts of the format spec) can be called on empty format specs or format specs that don't include that particular part. _Parse_arg_id has the validation since it's only called as more of a utility after we know there's an ID to parse.

Reply:
If it's called on an empty format spec it increments `_Begin` past `_End`??!?